Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -10,6 +10,18 @@
 ly_get_list_relative_pal_filename(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME})
 ly_get_list_relative_pal_filename(common_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/Common)
 
+ly_add_target(
+    NAME AzFramework.Headers
+    NAMESPACE AZ
+    HEADERONLY
+    INCLUDE_DIRECTORIES
+        INTERFACE
+            .
+            ${pal_dir}
+)
+
+include(${pal_dir}/platform_${PAL_PLATFORM_NAME_LOWERCASE}_window_system.cmake)
```

Comment:
Similar to the above comment, could this be moved into Code/Framework/AzFramework/Platform/Linux/platform_linux.cmake ?

Having it here means we'll have to add empty 'platform_{platform}_window_system.cmake' boilerplate files for every other platform we support.

Reply:
This is fixed now.